### ISSUE TITLE
Run oclint in parallel

### ISSUE THREAD

**Issue by ssmiech** [fastlane/fastlane#1443]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

Oclint can be really slow. It is relatively easy to make it run linting in parallel by grouping the files and runnig multiple `oclint` processes 
I would like to enable following oclint params: 
- parallel [boolean], false by default
- num_processes [integer], number of machine processors by default

Dependencies: 
- [Parallel gem](https://github.com/grosser/parallel)

**Comment by andybest** [comment]
This should be possible- the trick would be parsing the output of the separate processes and merging it back together. This would require quite a bit of work, since there are 6 output formats for OCLint

**Comment by TKBurner** [comment]
Thanks for taking a look at this @andybest. We'll leave this open in case any one wants to make a PR.

👍
